Makefile

```diff
@@ -214,6 +214,19 @@ endif
 	rm ./cmd/otelcol/translatesfx
 	rm ./cmd/otelcol/migratecheckpoint
 
+.PHONY: docker-otelcol-arm64
+docker-otelcol-arm64:
+ifneq ($(SKIP_COMPILE), true)
+	$(MAKE) binaries-linux_arm64
+endif
+	cp ./bin/otelcol_linux_arm64 ./cmd/otelcol/otelcol
+	cp ./bin/translatesfx_linux_arm64 ./cmd/otelcol/translatesfx
+	cp ./bin/migratecheckpoint_linux_arm64 ./cmd/otelcol/migratecheckpoint
+	docker build -t otelcol:arm64 --build-arg SMART_AGENT_RELEASE=$(SMART_AGENT_RELEASE) ./cmd/otelcol/
```

Comment:
We currently don't release the smart agent for arm64 (only amd64), so we shouldn't include it with the arm64 image.  I think we should create a new dockerfile, e.g. `Dockerfile.arm64`, that excludes the the smart agent stage and download/extraction, and update the command accordingly for the new dockerfile.

Reply:
The smart agent bundle is included for the `smartagent` [receiver](https://github.com/signalfx/splunk-otel-collector/tree/main/internal/receiver/smartagentreceiver) and [extension](https://github.com/signalfx/splunk-otel-collector/tree/main/internal/extension/smartagentextension) to collect metrics via smart agent monitors not yet available with the collector (there are notes for amd64 only support).  Also, see https://github.com/signalfx/splunk-otel-collector/blob/main/docs/signalfx-smart-agent-migration.md.

We can try building multi-arch images with a single dockerfile, but just thought it may be simpler with separate dockerfiles than handling conditional multi-stage commands based on the arch for the smart agent.